src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/MissingJavadocMethodCheck.java

```diff
@@ -129,6 +160,18 @@
  *   &lt;property name="excludeScope" value="protected"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *   private void test1() {} // violation, the private method is missing javadoc
+ *   &#47;**
+ *    * Some description here
+ *    *&#47;
+ *   private void test1() {} // ok
+ *   protected void test2() {} // ok
```

Comment:
ok -> OK ay both lines above
to make it consistent

Reply:
Done

Reply:
Done